biomaj_core/utils.py

```diff
@@ -448,6 +452,9 @@ def uncompress(archivefile, remove=True):
             if archivefile.endswith('.tar.gz'):
                 subprocess.check_call("tar xfz " + archivefile + " --overwrite -C " + os.path.dirname(archivefile), shell=True)
                 is_archive = True
+            elif archivefile.endswith('.tgz'):
```

Comment:
could be merged with above test as same tool/command, only check for different extensions

Reply:
done